include/fw_metadata.h

```diff
@@ -199,6 +201,36 @@ fw_firmware_info_get(u32_t firmware_address)
 	return NULL;
 }
 
+#define FW_INFO_OFFSET0 0x200
```

Comment:
This looks weird, what is this?

Reply:
Yep, will do. BTW I'd appreciate ideas here, like which offsets should be allowed, and how to implement it differently etc.

Reply:
Added comment